src/components/NavBar/NavBarModal.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import styles from './NavBarModal.module.scss';
+
+const NavBarModal = (props) => {
+    
+    return props.open ? ReactDOM.createPortal(
+        <div id="modal_navbar" className={styles.transparentModal} onClick={props.onClose}>
+        </div>,
+        document.body
```

Comment:
It's better to inject portals into a div alongside "root", rather than to the document's body. That would make them easier to find in the visual tree, therefore easier to debug.
The div should be added into `public/index.html` and you might call it "modals" or something along these lines.

Reply:
I added a div with the id "modals".